www/js/modules/dashboard.js

```diff
@@ -0,0 +1,1173 @@
+/* global $ */
+
+/* OpenSprinkler App
+ * Copyright (C) 2015 - present, Samer Albahra. All rights reserved.
+ *
+ * This file is part of the OpenSprinkler project <http://opensprinkler.com>.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// Configure module
+var OSApp = OSApp || {};
+OSApp.Dashboard = OSApp.Dashboard || {};
+
+OSApp.Dashboard.displayPage = function() {
+	// Display the home dasbhoard main view
+	var cards, siteSelect, currentSite, i, sites;
+	var page = $( "<div data-role='page' id='sprinklers'>" +
```

Comment:
Same comment, any chance we can restore the indentation? Hard to read without it. Can use a template string if it's easier.